data/competencies.yml

```diff
@@ -925,3 +925,15 @@
   level: senior1-to-senior2
   area: leadership
   domain: null
+
+
+- id: f173b7e9-1c46-4afd-a659-d264aa173759
+  summary: Contributes to inclusive organisational culture
+  examples:
+    - Membership of internal groups that promote diversity and inclusion (such as FT Embrace, FT Proud, Mental health first aiders)
```

Comment:
I'd be a bit wary of those examples as to me it feels both of them require a commitment outside 9-5 (standard working hours) and may clash with things like child care etc. It also feels the senior you are, you end up with more meetings, and those groups membership often means yet another meeting.

Reply:
Maybe have "ensure their reports and their fellow team members diversity needs are respected"? That phrasing would include stuff like having Retro snack be such that everyone can have something, or that person on crutches can sit down during a meeting and still see the presentation, or arranging meetings such that people who have regular obligations (medical appointments / child care) don't have clashes...

Reply:
Thanks Ania, we have tried to remove any expectation of things outside of your 9–5 in other competencies, but I think the example of being involved in these groups is important. I'd also expect work on this stuff to fit into someone's work day – contributing to the employee networks is still work. If somebody ends up doing work for FT Embrace one evening, for example, I'd expect them to manage their working hours for the rest of the week to make sure they weren't spending their own free time on it.

Do you have a suggestion of how we can phrase this differently? I'm happy with [Lily's suggestion](https://github.com/Financial-Times/engineering-progression/pull/329#discussion_r467818232) but it may be that there are further changes we could make.

> or that person on crutches can sit down during a meeting and still see the presentation, or arranging meetings such that people who have regular obligations (medical appointments / child care) don't have clashes...

I really like these examples, I think we should add something around these.

Reply:
I'm happy with Lily's suggestion! :)

Reply:
Being a member of an employee network is part of an employee's contracted hours, so it would not be anything 'extra'. This is why it's so important for it to be in the competency matrix, because it detracts from time that could be spent on other tasks (that do feature in the matrix) despite adding a huge amount of value to the business.  John Ridding has made it explicitly clear that this work _should not_ be done outside of work hours.

It is also not necessarily work that is 'another meeting' - although it's defo frustrating when it is, I agree Ania!! ;)

Both Keran and I were very eager to have the variety of skills we (and many others) have demonstrated and developed through this work to be recognised.  I agree with Lily's suggestion if the degree to which an employee contributes is proportional to the degree to which it weighs as evidence.

Reply:
Thanks for the extra context on employee networks, I didn't know that there was an explicit instruction from JR on work not being done out of hours.

On this:
> I agree with Lily's suggestion if the degree to which an employee contributes is proportional to the degree to which it weighs as evidence.

Competencies [are not weighted against each other](https://engineering-progression.ft.com/docs/faq/#are-competencies-weighted--are-some-competencies-more-important-than-others), and I don't think they really can be in order to work for all groups across the department.

Reply:
Could we please make sure we make it explicit that "Being a member of an employee network is part of an employee's contracted hours" - here and in other places where relevant (there's a "chat over a coffee with a prospective hire" in mid-senior, for example)? I think that was my issue - in various organisation this is handled differently, often on a case-by-case basis. This progression framework is both for current but also for prospective employees of the FT, and I always am for stating these sort of things explicitly. 🙂

Reply:
I don't think it's the job of the competencies to document this, as we'd be hard-coding other organisational information in them and it could easily become out of date. We've taken the approach so far of deferring to other documentation from the FT on this kind of thing as it's not our responsibility to make sure FT employees know this stuff.

We currently do this via Supporting URLs (e.g. https://github.com/Financial-Times/engineering-progression/blob/master/data/competencies.yml#L73) so we could add a link out to documentation on employee networks from this competency.

Another thing we could do here (a PR would be more than welcome) is adding something to [the FAQ](https://engineering-progression.ft.com/docs/faq/). Additional work outside of your contracted hours is 100% not a requirement for any of the competencies, I'm happy to make that fact more explicit in a central place rather than updating each of the examples.

Reply:
Some great news on this today, we just got handed [the perfect document](https://docs.google.com/document/d/1W9GKyBEkSpj6iKztu2tzKIetbeeo0smZQrBez9SlVqM/edit) to link to on employee networks and the time you can claim back on them. I think we should make this a supporting URL on this competency

Reply:
We need to be careful here re: this document, as it's for people formally serving on the committees.  Attending events still should be done within working hours, and volunteering to help for a one-off specific event would count too I expect, if that help was 'formally' acknowledged by the relevant committee.

Reply:
I think this is what I meant when I said about weighting - attending an event shows good intent to learn and develop (and therefore contribute professionally to an inclusive culture etc), but in itself should not be enough to 'tick that box' without the explicit follow up behavioural evidence.

Reply:
OK some actions from this thread:

1. ~Add an item to the FAQ to explain that we don't expect work out of hours~ #330 thanks @jenniferemshepherd!
2. Add some supporting URLs for this competency which outline how the employee networks work. We already have this [document on claiming back 6 days](https://docs.google.com/document/d/1W9GKyBEkSpj6iKztu2tzKIetbeeo0smZQrBez9SlVqM/edit), and we think that the [internal Employee Networks page is another good one to include](https://sites.google.com/ft.com/inside/hr-talent-learning/diversity-inclusion/employee-networks). @jenniferemshepherd and @AniaMakes are you happy with these as actions?